package.json

```diff
@@ -46,49 +46,51 @@
     "network info"
   ],
   "peerDependencies": {
-    "react-native": ">=0.59"
+    "react-native": ">=0.65.1"
```

Comment:
peerDependency verification is getting much stricter in npm7
I don't think our intention is to require >0.65.1 is it? Did your changes actually drop compatibility with >0.59?
The implication of your peerDependency change here is that this package only works with 0.65.1 and higher and you want installs on npm7 to fail by default. I don't think that is actually your intention?

Reply:
That is actually my intention.
 
 Since 0.65.1 react-native don't use jcenter. 
 
I gause, fail while install on npm7 can helps people see real trouble with jcenter.
 
 They will begin to figure out why it failed and see that "dark times" are coming and that react-native needs to be updated 
 
 Or you project builds will no longer be able to resolve artifacts from JCenter after February 1st, 2022 -> all builds fail -> loss money -> everyone sad

Reply:
You are wrong about dark times in the future, I already posted the link for you indicating JCenter will be up for module resolution indefinitely. There is no rush, there will be no lost money or sadness, please don't be dramatic.

I understand react-native 0.65.1+ does not use Jcenter. Similar to how I patched the AndroidX transition for everyone with https://github.com/mikehardy/jetifier I was preparing to do the same with https://github.com/mikehardy/jcenterizer/ when I saw the news JCenter would stay up.

So this is not urgent.

Thus I repeat my most important question: Did your changes actually drop compatibility with >0.59?

My read of your PR is that you have not actually dropped compatibility. Thus it's user-unfriendly to do what you've done here with the peerDependency change.

I do not support this change.

Reply:
>You are wrong about dark times in the future, I already posted the link for you indicating JCenter will be up for module resolution indefinitely. There is no rush, there will be no lost money or sadness, please don't be dramatic.

Okay, maybe I exaggerating a little bit:D 

I just wanted to nudge people using this library to pay attention to jcenter:o
I guess it will helps everyone updated faster and there are no problems later 😅 

I returned peerDependencies as it was before me
```
"peerDependencies": {
    "react-native": ">=0.59"
}
```